doc/tutorials/partition-mode.rst

```diff
@@ -0,0 +1,332 @@
+        UOS VM kernel. First and third entries should total to ``mem_size``.
+
+        Rest of the memory between ``mem_size`` and start of PCI hole i.e.
+        0xC0000000 should go into fourth entry. Reference code is for 512MB
+        for each UOS VM. If the memory requirement for each UOS VM is 1GB
+        instead, ``e820_default_entries`` looks like below:
+
+         .. code-block:: none
+
+            const struct e820_entry e820_default_entries[NUM_E820_ENTRIES] = {
+                    {   /* 0 to mptable */
+                        .baseaddr =  0x0U,
+                        .length   =  0xEFFFFU,
+                        .type     =  E820_TYPE_RAM
+                    },
+
+                    {   /* mptable 65536U */
+                        .baseaddr =  0xF0000U,
+                        .length   =  0x10000U,
+                        .type     =  E820_TYPE_RESERVED
+                    },
+
+                    {   /* mptable to lowmem */
+                       .baseaddr =  0x100000U,
+                       .length   =  0x3FF00000U,
+                       .type     =  E820_TYPE_RAM
+                    },
+
+                    {    /* lowmem to PCI hole */
+                        .baseaddr =  0x40000000U,
+                        .length   =  0x80000000U,
+                        .type     =  E820_TYPE_RESERVED
+                    },
+
+                    {   /* PCI hole to 4G */
+                        .baseaddr =  0xe0000000U,
+                        .length   =  0x20000000U,
+                        .type     =  E820_TYPE_RESERVED
+                    },
+                }
+
+
+   E.   Do modifications to ``mptable_vm(n)`` (where n stands for each UOS VM). Each UOS
+        VM has a mptable defined in ``arch/x86/guest/mptable.c``.
+
+         Make sure ``VM1_NUM_CPUS`` and ``VM2_NUM_CPUS`` are defined to the same
+         values as they are in ``partition\vm_description.c``.
+
+         ``proc_entry_array`` member in struct ``mptable_info`` is a flexible array.
+         Each cpu of the UOS should be represented by a ``proc_entry`` in ``proc_entry_array``.
+
+        ``apic_id`` in the ``proc_entry`` should be same as physical APIC ID
+        of physical CPU that belongs to the UOS VM.
+
+        For the following configuration defined in ``partition/vm_description.c``,
+
+               .. code-block:: none
+
+                  #define VM1_NUM_CPUS    4U
+
+                  /* Logical CPU IDs assigned to this VM */
+                  uint16_t VM1_CPUS[VM1_NUM_CPUS] = {0U, 2U, 4U, 6U};
+
+         ``proc_entry_array`` in ``mptable_vm1`` should have 4 ``proc_entry``
+         members and the ``apic_id`` of each ``proc_entry`` should be corresponding
+         to the physical APIC id of the physical processors ``{0U, 2U, 4U, 6U}``.
+
+Building ACRN with partition mode enabled
+*****************************************
+
+For MRB board, change directory to ``acrn-ebtool`` (the top-level directory where
```

Comment:
Remove references to MRB board, and verify instructions work for the open source supported platforms.